From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@4c74356b41 - Thank you!

v-alje committed:
Bringing even with master.

kgremban committed:
Cleaning up extranneous notes in includes

kgremban committed:
Acrolinx pass

kgremban committed:
Add include, csharp-node through devguide-messages-c2d

PRMerger15 committed:
Merge pull request #34830 from kgremban/iotbasic_addinclude

Add include, csharp-node through devguide-messages-c2d

PRMerger16 committed:
Merge pull request #34812 from kgremban/iotbasic_cleanup

Cleaning up extraneous notes in includes

kgremban committed:
Add include, device-management thru java-java

kgremban committed:
Merge branch 'release-iotbasic' of https://github.com/microsoft/azure-docs-pr into iotbasic_addinclude2

PRMerger12 committed:
Merge pull request #34838 from kgremban/iotbasic_addinclude2

Add include, device-management thru java-java

v-alje committed:
Bringing even with master.

kgremban committed:
Add include, node thru python

PRMerger19 committed:
Merge pull request #34926 from kgremban/iotbasic_include

Add include, node thru python

kgremban committed:
Add partial include, devguide

kgremban committed:
Moved include

Commit:
Merge pull request #34930 from kgremban/iotbasic-include2

Add partial include, devguide

kgremban committed:
Add include, device-management thru mqtt

Commit:
Merge pull request #34948 from kgremban/iotbasic-include3

Add include, device-management thru mqtt

kgremban committed:
Add examples to partial include

v-alje committed:
Resolving merge conflict

PRMerger19 committed:
Merge pull request #34997 from kgremban/iotbasic-updateinclude

Add examples to partial include

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Commit:
add new Labs section & update links

Commit:
update links

v-alje committed:
Bringing even with master.

kgremban committed:
Update iot/event hubs comparison

v-alje committed:
Bringing even with master.

kgremban committed:
Updates and renaming scale your solution

kgremban committed:
Update iot-hub-scaling.md

rjagiewich committed:
Merge pull request #35365 from kgremban/iotbasic-scaling

Updates and renaming scale your solution

v-alje committed:
Bringing even with master.

kgremban committed:
New article - how to upgrade IoT Hub

Commit:
update icons

kgremban committed:
Added next steps section

ktoliver committed:
Merge pull request #35390 from kgremban/iotbasic-upgrade

New article - how to upgrade IoT Hub

kgremban committed:
Changed link text in IoT includes

Commit:
Merge pull request #35430 from kgremban/iotbasic-Afeedback

Changed link text in IoT includes

kgremban committed:
A's feedback on scaling

v-alje committed:
Bringing even with master.

kgremban committed:
Update iot-hub-scaling.md

PRMerger17 committed:
Merge pull request #35434 from kgremban/iotbasic-Afeedback

A's feedback on scaling

kgremban committed:
IoT Basic updates to quotas and throttle

kgremban committed:
Merge branch 'release-iotbasic' of https://github.com/microsoft/azure-docs-pr into iotbasic-compareeventhub

kgremban committed:
Update iot-hub-devguide-quotas-throttling.md

Commit:
Merge pull request #35457 from kgremban/iotbasic-quota

IoT Basic updates to quotas and throttle

kgremban committed:
More changes

kgremban committed:
Adding text about free tier for evaluation

PRMerger16 committed:
Merge pull request #35472 from kgremban/iotbasic-scaling

Adding text about free tier for evaluation

kgremban committed:
Adding basic messaging to edge and deps

kgremban committed:
Update hub creation docs for basic tier

Commit:
Merge pull request #35486 from kgremban/iotbasic-edgedps

Adding basic messaging to edge and deps

v-alje committed:
Bringing even with master.

Commit:
Merge pull request #35489 from kgremban/iotbasic-getstarted

Update hub creation docs for basic tier

kgremban committed:
Create include for DPS basic note

v-alje committed:
Bringing even with master.

dhirajgandhi committed:
Added Partner Admin Link article

v-shils committed:
Merge pull request #35537 from kgremban/iotbasic-dps

Create include for DPS basic note

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

Lisaco88 committed:
edit pass: hana-overview-high-availability-disaster-recovery

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

v-alje committed:
Bringing even with master.

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

kgremban committed:
Article updates and checkmark image

Ja-Dunn committed:
Merge pull request #35328 from kgremban/iotbasic-compareeventhub

Update iot/event hubs comparison

v-alje committed:
Bringing even with master.

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

kgremban committed:
Ashita's feedback on comparison

PRMerger19 committed:
Merge pull request #35916 from kgremban/iotbasic-compare

Ashita's feedback on comparison

v-alje committed:
Bringing even with master.

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

v-alje committed:
Bringing even with master.

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hana-overview-high-availability-disaster-recovery

v-alje committed:
Bringing even with master.

kgremban committed:
Updating referenes to different tier levels

rjagiewich committed:
Merge pull request #36089 from kgremban/iotbasic-Afeedback

Updating referenes to different tier levels

v-alje committed:
Bringing even with master.

Lisaco88 committed:
edit pass: hana-overview-high-availability-disaster-recovery

v-alje committed:
Bringing even with master.

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hana-overview-high-availability-disaster-recovery

v-mavick committed:
Migrating door-monitor.md to MS Docs.

v-alje committed:
Bringing even with master.

Lisaco88 committed:
edit pass: hana-overview-high-availability-disaster-recovery

v-mavick committed:
Corrected text per reviewer feedback. Replaced sendgrid-manage.png.

ShawnJackson committed:
Fixing validation errors

ShawnJackson committed:
Update hana-overview-high-availability-disaster-recovery.md

v-alje committed:
Bringing even with master.

ShawnJackson committed:
Update hana-overview-high-availability-disaster-recovery.md

kgremban committed:
Basic updates from bugbash

dstrockis committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into acs-migration-2

v-alje committed:
Bringing even with master.

dstrockis committed:
Adding Sharepoint section, updating some dates

kgremban committed:
More feedback to scaling and upgrade

kgremban committed:
Update iot-hub-scaling.md

PRMerger12 committed:
Merge pull request #36275 from kgremban/iotbasic-bugbash

Basic updates from bugbash

v-alje committed:
Bringing even with master.

dstrockis committed:
Edits based on SharePoint feedback

kgremban committed:
Added unit info to scaling doc

Commit:
Merge pull request #36435 from kgremban/iotbasic-bugbash

Added unit info to scaling doc

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

JeffGoldner committed:
Fix confusing root cert guidance

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

v-alje committed:
Bringing even with master.

DavidBerg-MSFT committed:
Update Linux Coremark Benchmark numbers

Update the Linux Coremark benchmark numbers with the latest data and newest SKUs.

DavidBerg-MSFT committed:
Add note regarding benchmark scores being updated

Windows SPEC INT numbers are being updated, but will take a little while.  In the meantime we want to direct people to the Linux Coremark numbers which are more up to date and contain more VM sizes.

v-alje committed:
Bringing even with master.

DavidBerg-MSFT committed:
Add Std Dev %

Standard Deviation as a % of average is a convenient way to compare variation between different SKUs.

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

kgremban committed:
Remove basic references in cli docs

douglaslMS committed:
New article from PM with screenshots, TOC update.

MikeRayMSFT committed:
Add script.

MikeRayMSFT committed:
Improvements for acrolinx

MikeRayMSFT committed:
Update date.

douglaslMS committed:
Fixed issues identified by PR reviewer.

v-alje committed:
Bringing even with master.

barbkess committed:
new topic - upgrade to gen2

barbkess committed:
fix manager

barbkess committed:
fix

v-alje committed:
Bringing even with master.

PRMerger20 committed:
Merge pull request #36641 from kgremban/iotbasic-cli

Remove basic references in cli docs

cherylmc committed:
format

v-alje committed:
Bringing even with master.

cherylmc committed:
format

rachel-msft committed:
georestore ga cli

juliekohler committed:
Create supportedLanguages.md

juliekohler committed:
Update supportedLanguages.md

ShawnJackson committed:
edit pass: active-directory-b2c-reference-language-customization

juliekohler committed:
Update supportedLanguages.md

ShawnJackson committed:
Acrolinx fixes

ajlam committed:
Merge pull request #4 from MicrosoftDocs/master

Update fork

juliekohler committed:
Update supportedLanguages.md

juliekohler committed:
Update toc.yml

v-alje committed:
Bringing even with master.

ajlam committed:
update metadata

DavidBerg-MSFT committed:
Update compute-benchmark-scores.md

rachel-msft committed:
extension

rachel-msft committed:
correct mysql links

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grestorecli

rachel-msft committed:
Update howto-restore-server-cli.md

rachel-msft committed:
update extension version

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

dhirajgandhi committed:
Update billing-partner-admin-link-started.md

cwatson-cat committed:
Updated supported browser topic to add link to mobile app

MikeRayMSFT committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 20180329_GrantNTAuthoritySystemPermissionsForAG

MikeRayMSFT committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 20180329_GrantNTAuthoritySystemPermissionsForAG

barbkess committed:
fix warning

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#6574

Updates link to point to the Windows version of SSH key creation, not Mac/Linux

mrbullwinkle committed:
small pricing changes

Brenduns committed:
relnote and other updates

PRMerger13 committed:
Merge pull request #36327 from dstrockis/acs-migration-2

Update ACS Migration Doc with Sharepoint section

PRMerger14 committed:
Merge pull request #36829 from iainfoulds/patch-5

[VM] Linux PowerShell quickstart

JeffGoldner committed:
Add Veeam backup&replication

JeffGoldner committed:
adjust date

Brenduns committed:
relnote and other updates

PRMerger16 committed:
Merge pull request #36831 from mrbullwinkle/mrb_04_02_2018_pricing_changes

small pricing changes

JeffGoldner committed:
Merge branch 'master' into master

PRMerger17 committed:
Merge pull request #36792 from ajlam/master

Update metadata

BryanLa committed:
replace/modify the Extract security artifacts section

Brenduns committed:
relnote update

douglaslMS committed:
Added clarification per customer suggestion, cleaned up titles.

Brenduns committed:
udpate version

cherylmc committed:
Update point-to-site-how-to-vpn-client-install-azure-cert.md

Brenduns committed:
udpate version

cherylmc committed:
linux

Commit:
Merge pull request #36835 from douglaslMS/sync-titles-issue

Added clarification per customer suggestion, cleaned up titles.

Commit:
fixing screenshot and updating text for clarity

roygara committed:
Updating article owners.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into reassignOwner

PRMerger14 committed:
Merge pull request #36832 from Brenduns/april-02

relnote and other updates

rachel-msft committed:
remove sync

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grestorecli

rachel-msft committed:
Merge branch 'grestorecli' of https://github.com/rachel-msft/azure-docs-pr into grestorecli

rloutlaw committed:
updating functions content with changes from user study

v-alje committed:
Bringing even with master.

v-shils committed:
Merge pull request #36836 from cherylmc/patch-1

Update point-to-site-how-to-vpn-client-install-azure-cert.md

dhirajgandhi committed:
Update TOC.md

PRMerger18 committed:
Merge pull request #36838 from roygara/reassignOwner

Reassigning article ownership so that issues come to me, as intended

PRMerger19 committed:
Merge pull request #36827 from cwatson-cat/4-2-portal-upd

Updated supported browser topic to add link to mobile app

v-shils committed:
Merge pull request #36837 from Blackmist/freshness

fixing screenshot and updating text for clarity

cherylmc committed:
link

mahesh-unnikrishnan committed:
Update active-directory-ds-getting-started-password-sync.md

Commit:
Merge pull request #36803 from rachel-msft/updatev

update extension version

v-shils committed:
Merge pull request #36741 from cherylmc/toc2

format

mahesh-unnikrishnan committed:
Update active-directory-ds-getting-started-password-sync.md

Commit:
Merge pull request #36842 from rloutlaw/rloutlaw/azure-functions-fixes

updating functions content with changes from user study

GitHubber17 committed:
Merge pull request #36529 from JeffGoldner/master

Fix confusing root cert guidance

Ja-Dunn committed:
Merge pull request #35744 from Lisaco88/hana-overview-high-availability-disaster-recovery

edit pass: hana-overview-high-availability-disaster-recovery

douglaslMS committed:
Replaced GUIDs with subscription ID placeholder text.

Commit:
Task 1045: Add Premium to Verizon product name for DSA note

GitHubber17 committed:
Merge pull request #36839 from cherylmc/swan

linux

GitHubber17 committed:
Merge pull request #36843 from mahesh-unnikrishnan/patch-17

Clarify that password hashes are synced to AAD-DS.

BryanLa committed:
heavy mods to Extract the security artifacts and Create the device registration software sections

BryanLa committed:
heavy mods to Extract the security artifacts and Create the device registration software sections

v-shils committed:
Merge pull request #35224 from v-savila/1202094

add new Labs section & update links

cynthn committed:
Update attach-disk-ps.md

douglaslMS committed:
Updated all MI references for public preview.

DavidBerg-MSFT committed:
Changed Note formatting per standard.

PRMerger17 committed:
Merge pull request #36845 from dksimpson/dean-950

Task 1045: Add Premium to Verizon product name for DSA note

mahesh-unnikrishnan committed:
Merge branch 'master' into patch-18

PRMerger19 committed:
Merge pull request #36846 from cynthn/patch-73

Update attach-disk-ps.md

PRMerger19 committed:
Merge pull request #36847 from douglaslMS/adf-fix-mi

Updated all MI references for public preview.

GitHubber17 committed:
Merge pull request #36644 from douglaslMS/adf-cont-integration2

New article from PM with screenshots, TOC update.

barbkess committed:
ms.services

PRMerger20 committed:
Merge pull request #36844 from mahesh-unnikrishnan/patch-18

Clarify that password hashes are synced to AAD-DS.

twooley committed:
[Azure AD B2B] Rewrite of existing article

Commit:
Merge pull request #36645 from MikeRayMSFT/20180329_GrantNTAuthoritySystemPermissionsForAG

Add script to set account permissions

MikeRayMSFT committed:
Update virtual-machines-windows-portal-sql-create-failover-cluster.md

rachel-msft committed:
heading

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grestorecli

PRMerger13 committed:
Merge pull request #36854 from MikeRayMSFT/patch-6

Update virtual-machines-windows-portal-sql-create-failover-cluster.md

Ja-Dunn committed:
Merge pull request #36683 from DavidBerg-MSFT/patch-1

Update Coremark (Linux) Benchmark Scores

Ja-Dunn committed:
Merge pull request #36852 from twooley/twadminb2b3

[Azure AD B2B] Rewrite of existing article

rachel-msft committed:
Update howto-restore-server-cli.md

Ja-Dunn committed:
Merge pull request #36750 from barbkess/gen2

Gen2

rachel-msft committed:
Update howto-restore-server-cli.md

juliekohler committed:
Update and rename supportedLanguages.md to supported-languages.md

juliekohler committed:
Update toc.yml

BryanLa committed:
tweaks

rachel-msft committed:
fix typos

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grestorecli

maertendMSFT committed:
Updating versions

juliekohler committed:
Update supported-languages.md

juliekohler committed:
Update supported-languages.md

ktoliver committed:
Merge pull request #35551 from dhirajgandhi/dhgandhi_changes

Added Partner Admin Link article

PRMerger13 committed:
Merge pull request #36860 from maertendMSFT/tools

Updating versions

v-mavick committed:
Corrected text for the H1 Title element.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

BryanLa committed:
tweaks

Ja-Dunn committed:
Merge pull request #36856 from rachel-msft/grestorecli

Geo restore cli

v-alje committed:
Merge pull request #36863 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #36791 from ShawnJackson/active-directory-b2c-reference-language-customization

edit pass: active-directory-b2c-reference-language-customization

ktoliver committed:
Merge pull request #36187 from v-mavick/door-monitor

Migrating door-monitor.md to MS Docs.

ktoliver committed:
Merge pull request #36833 from BryanLa/bryanla-dps-tutorial-fixes

Replace/modify the Extract security artifacts section in DPS tutorial

v-alje committed:
Merge pull request #36710 from MicrosoftDocs/release-iotbasic

Release iotbasic

GitHubber17 committed:
Merge pull request #36788 from juliekohler/patch-1

Create supportedLanguages.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
